DataStructures.Tests/BinaryHeapTests.cs

```diff
@@ -0,0 +1,182 @@
+﻿using System;
+using System.Collections.Generic;
+
+using NUnit.Framework;
+
+namespace DataStructures.Tests
+{
+    class BinaryHeapTests
```

Comment:
Codacy found an issue: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=6375083)